### PR TITLE
move staticip from tp to ga

### PR DESCRIPTION
**Pull request by sgaoshang**
Set vSphere staticip job to GA

### CODE REVIEW

**Comment by sgaoshang** [comment]
/pj-rehearse periodic-ci-openshift-openshift-tests-private-release-4.16-amd64-nightly-vsphere-ipi-static-ip-tp-f28

**Comment by sgaoshang** [comment]
@jianlinliu Hi, pls help to review it, thanks.

File: ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.16__amd64-nightly.yaml [hunk 1]
```diff
@@ -2722,17 +2722,13 @@ tests:
   cron: 44 0 14 * *
   steps:
     cluster_profile: vsphere-connected-2
-    env:
-      FEATURE_SET: TechPreviewNoUpgrade
     test:
     - chain: openshift-e2e-test-qe-automated-release
     workflow: cucushift-installer-rehearse-vsphere-ipi-static-ip
 - as: vsphere-ipi-static-ip-tp-f28-destructive
```
Comment (jianlinliu) [inline ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.16__amd64-nightly.yaml:2728]:
```suggestion
- as: vsphere-ipi-static-ip-f28-destructive
```

Comment (jianlinliu) [reply ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.16__amd64-nightly.yaml:2728]: BTW, can you also add a z-stream upgrade job in openshift-openshift-tests-private-release-4.16__amd64-nightly-4.16-upgrade-from-stable-4.16.yaml

**Comment by jianlinliu** [comment]
cc @liangxia to review.

**Comment by sgaoshang** [comment]
/bj-rehearse periodic-ci-openshift-openshift-tests-private-release-4.16-amd64-nightly-vsphere-ipi-static-ip-f28

**Pull request merged**